### PR TITLE
Added quotes categories + moved quotes to JSON storage.

### PR DESCRIPTION
**Pull request by Aquerr**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -41,4 +41,4 @@ hs_err_pid*
 src/main/java/io/github/aquerr/futrzakbot/secret
 /futrzaki
 /config.conf
-/quotes.txt
\ No newline at end of file
+/quotes.json
```
Comment (Shimozuke) [inline .gitignore:44]: txt better

File: src/main/java/io/github/aquerr/futrzakbot/command/CommandManager.java [hunk 2]
```diff
@@ -23,7 +23,7 @@
 
 public class CommandManager
 {
-    public static final String COMMAND_PREFIX = "!t";
+    public static final String COMMAND_PREFIX = "!f";
```
Comment (Shimozuke) [inline src/main/java/io/github/aquerr/futrzakbot/command/CommandManager.java:26]: !t better

**Review by Shimozuke — APPROVED** [review]

File: src/main/java/io/github/aquerr/futrzakbot/command/QuoteCommand.java [hunk 3]
```diff
@@ -39,4 +114,29 @@ public String getDescription()
     {
         return "Wylosuj cytat";
     }
+
+    @Override
+    public String getUsage()
+    {
+        return Command.super.getUsage() + "\n" + "Use ? to show available categories.";
```
Comment (Shimozuke) [inline src/main/java/io/github/aquerr/futrzakbot/command/QuoteCommand.java]: A gdzie CommandSource

File: src/main/java/io/github/aquerr/futrzakbot/command/QuoteCommand.java [hunk 4]
```diff
@@ -1,24 +1,99 @@
...
+        TextChannel textChannel = context.getTextChannel();
+        Member member = context.getMember();
+
+        if ("?".equals(category))
+        {
+            // print help + possible categories
+            List<QuoteCategory> categories = this.quoteGame.getAvailableCategories();
+            printAvailableCategories(textChannel, categories);
+        }
+        else if ("".equals(category))
```
Comment (Shimozuke) [inline src/main/java/io/github/aquerr/futrzakbot/command/QuoteCommand.java:57]: ?

Comment (Aquerr) [reply src/main/java/io/github/aquerr/futrzakbot/command/QuoteCommand.java:57]: O co chodzi?

**Pull request merged**